content/en/blog/argo-insurance-workflow/index.md

```diff
@@ -0,0 +1,126 @@
++++
+title = "Fission Function Orchestration with Argo Workflows"
+date = "2022-09-21T10:15:00+05:30"
+author = "Neha Gupta"
+description = "Multiple Fission function orchestration using Argo Workflows"
+categories = ["Tutorials"]
+type = "blog"
+images = ["images/featured/fission-zapier-featured.png"]
```

Comment:
This image here is the header image for the blog post (_to test, just share any of the last 5-6 blog post to twitter and you'll see it)_ Currently this points to zapier-featured image.

This needs to be changed to a relevant image, I'll create one and share with you that you can update.

Reply:
That would help @techmaharaj